src/jobs.rs

```diff
@@ -93,13 +93,25 @@ fn generate_flatpakref(ref_name: &String,
         format!("{} from {}", app_id, reponame)
     };
 
+    /* We only want to deploy the collection ID if the flatpakref is being generated for the main
+     * repo not a build repo.
+     */
+    let deploy_collection_id_config = match &repoconfig.collection_id {
+        Some(collection_id) => if repoconfig.deploy_collection_id && maybe_build_id == None {
+            format!("DeployCollectionID={}\n", collection_id)
+        } else {
+            "".to_string()
+        }
```

Comment:
Shouldn't there be a comma here?

Reply:
Yes, but I also noticed the pattern a few lines below of adding optional keys which seems simpler so I changed `DeployCollectionID` to work that way.